docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -37,6 +37,26 @@ image::images/additional-filters.png[Alerts table with Additional filters menu h
 
 * View detection alerts generated by a specific rule. Go to *Manage* -> *Rules*, then select a rule name in the table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule, including alerts from any previous or deleted revision of that rule.
 
+[float]
+[[group-alerts]]
+=== Group alerts
+
+beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
+
+[role="screenshot"]
+image::images/group-alerts.png[Alerts table with Group alerts by drop-down]
+
+Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The type of information displayed varies depending on the field used for grouping.
+
+To interact with grouped alerts:
+
+* Select *Take actions* to perform a bulk action on all alerts in a group, such as <<detection-alert-status,changing their status>>.
```

Comment:
I'll leave this comment once for wherever we mention `Select Take actions`.  By UX standards, it's technically a menu. 
```suggestion
* Select the *Take actions* menu to perform a bulk action on all alerts in a group, such as <<detection-alert-status,changing their status>>.
```